cmd/extproc/main.go

```diff
@@ -17,6 +20,27 @@ import (
 )
 
 func main() {
+	// Run the pprof server if the DISABLE_PPROF environment variable is not set.
+	//
+	// Enabling the pprof server by default helps with debugging performance issues in production.
+	// The impact should be negligible when the actual pprof endpoints are not being accessed.
+	if _, ok := os.LookupEnv("DISABLE_PPROF"); !ok {
+		go func() {
+			const pprofPort = "6060" // The same default port as in th Go pprof documentation.
+			mux := http.NewServeMux()
+			mux.HandleFunc("/debug/pprof/", pprof.Index)
+			mux.HandleFunc("/debug/pprof/cmdline", pprof.Cmdline)
+			mux.HandleFunc("/debug/pprof/profile", pprof.Profile)
+			mux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
+			mux.HandleFunc("/debug/pprof/trace", pprof.Trace)
```

Comment:
Does it make sense to add something like this, to enable profiles like `heap`, `goroutines`, etc?

```go
mux.HandleFunc("/debug/pprof/{profile}", func(w http.ResponseWriter, r *http.Request) {
    pprof.Handler(r.PathValue("profile")).ServeHTTP(w, r)
})
```